<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QDialog" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="result_disp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	property-alignment: 'AlignVCenter | AlignRight';
	border-bottom: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AC">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sign">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_percent">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plus">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mult">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_min">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_div">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;[x]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dot">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
